### PR TITLE
Fix #13: Add merge_sort.cpp

### LINKED ISSUE #13: Implement Merge Sort
Implement [merge sort algorithm](https://en.wikipedia.org/wiki/Merge_sort
) in C++.

Input:
An array of integers.
Output:
print the sorted array.

### PR DESCRIPTION
**Pull request by Pankaj729**

### CODE REVIEW

File: merge_sort.cpp [hunk 1]
```diff
@@ -0,0 +1,66 @@
+#include<bits/stdc++.h>
```
Comment (abarya) [inline merge_sort.cpp]: Is this required?

Comment (Pankaj729) [reply merge_sort.cpp]: ohh okay

**Comment by abarya** [comment]
Great!!

File: merge_sort.cpp (hunk 1 above)
Comment (anubhakushwaha) [reply merge_sort.cpp]: :P

**Pull request merged**